tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -3976,44 +3757,102 @@ Status ConvertGather(OpConverterParams* params) {
   return Status::OK();
 }
 
-Status ConvertMatMulHelper(OpConverterParams* params,
-                           TRT_TensorOrWeights tensor_input,
-                           TRT_ShapedWeights weights_raw, bool transpose_weight,
-                           string node_name) {
-  if (!tensor_input.is_tensor()) {
-    return errors::InvalidArgument("Input 0 expects tensor");
+Status ConvertFCHelper(OpConverterParams* params, nvinfer1::ITensor* tensor_a,
```

Comment:
Would you please split the MatMul change to another PR? It's self-contained and putting it here makes it hard to review this PR.
Thanks.